packages/firebase_admob/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.1
+
+* Fix listeners
```

Comment:
Could we make this `Fixed Dart 2 type errors.` or some such? We try to stick to whole sentences in past tense in the change log.